packages/front-end/components/Experiment/VariationsTable.tsx

```diff
@@ -93,104 +99,157 @@ const VariationsTable: FC<Props> = ({
-                <th
-                  className={`variation with-variation-border-bottom variation${i} pt-0 pb-1 align-bottom font-weight-normal`}
-                  style={{ borderTop: 0 }}
-                  key={i}
-                  scope="col"
-                >
-                  {hasDescriptions ? <div>{v.description}</div> : null}
-                  {hasUniqueIDs ? (
-                    <code className="small">ID: {v.key}</code>
-                  ) : null}
-                  {experiment.type !== "multi-armed-bandit" &&
-                  percentages?.[i] !== undefined ? (
-                    <div className="text-right text-muted">
-                      Split: {percentages[i].toFixed(0)}%
-                    </div>
-                  ) : null}
-                </th>
-              ))}
-            </tr>
-          </thead>
+        <Text size="8">
+          {canEditExperiment ? <PiCameraPlusLight /> : <PiCameraLight />}
+        </Text>
+      </Flex>
+    );
+  };
 
-          <tbody>
-            <tr>
-              {variations.map((v, i) => (
-                <td
-                  key={i}
-                  className={`align-middle ${canEditExperiment ? "pb-1" : ""}`}
-                  style={{
-                    minWidth: "17.5rem",
-                    maxWidth: "27rem",
-                    width: `${80 / Math.min(variations.length || 1, 4)}rem`,
-                    height: "inherit",
-                    borderBottom: canEditExperiment ? 0 : undefined,
-                  }}
-                >
-                  <div className="d-flex justify-content-center align-items-center flex-column h-100">
-                    {v.screenshots.length > 0 ? (
-                      <ScreenshotCarousel
-                        key={i}
-                        index={i}
-                        variation={v}
-                        canEditExperiment={canEditExperiment}
-                        experiment={experiment}
-                        mutate={mutate}
-                        maxChildHeight={200}
-                      />
+  return (
+    <Box mx="4">
+      <Flex gap={gap} wrap="wrap" direction={{ initial: "column", sm: "row" }}>
+        {variations.map((v, i) => (
+          <Box
+            key={i}
+            p="5"
+            pb="3"
+            flexGrow="0"
+            flexShrink="1"
+            flexBasis={{
+              // This might be a bit confusing, but 'gap' in flex box is not included in the flex basis width,
+              // which means percentages can't just be a simple division.
+              // This checks and does the math to make it fix perfectly
+              initial: `calc(100% / ${cols} - var(--space-${gap}) / ${cols} * (${cols} - 1))`,
+              sm: `calc(100% / ${cols - 1} - var(--space-${gap}) / ${
+                cols - 1
+              } * (${cols - 1} - 1))`,
+              md: `calc(100% / ${cols} - var(--space-${gap}) / ${cols} * (${cols} - 1))`,
```

Comment:
Can we use the `<Grid>` component instead of Flex to do this?  Seems to fit better with what we want here.  https://www.radix-ui.com/themes/docs/components/grid

Reply:
done